packages/next/src/export/index.ts

```diff
@@ -399,7 +399,7 @@ export async function exportAppImpl(
     disableOptimizedLoading: nextConfig.experimental.disableOptimizedLoading,
     // Exported pages do not currently support dynamic HTML.
     supportsDynamicHTML: false,
-    crossOrigin: nextConfig.crossOrigin || '',
+    crossOrigin: nextConfig.crossOrigin || undefined,
```

Comment:
```suggestion
    crossOrigin: nextConfig.crossOrigin,
```

I'm not sure if this is intended but your code was coercing the empty string to undefined. Perhaps we should leave it as is?